### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by yuunlee**
Add a README.md file and comments within .py

### CODE REVIEW

**Comment by cdolfi** [comment]
@yuunlee Can you add instructions as well on how to run your app?

File: models/density_metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+[AUGUR](https://github.com/chaoss/augur)
+
+## Code
+[GITHUB REPO]()
+
+## Metrics
+### Activity (-> weight)
+The activity metrics aim to discover the repository activeness percentage within an organization. The features taken into the calculation are the increment in star, watch, fork, issue, PR, open PR, closed PR, commit PR, commit, and committer. By taking the above monthly average activities and comparing them with last month to determine the increase or decrease in the attribute.
+
+Since different features indicate different importance of contribution, the following weights are given based on [ORBIT MODEL](https://orbitmodel.com/) and domain input.
```
Comment (JamesKunstle) [inline models/density_metrics/README.md:28]: Does the Orbit/Love model describe their reasoning for these weight values?

Clarity is essential here. Why does opening a pull request earn a 1.2 rather than a 1.5 for closing a pull request? Why are they not equal or 1.3 and 1.456 respectively? 

If their rationale isn't clear that doesn't invalidate your findings but it's essential to describe that.

File: models/density_metrics/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# DENSITY METRICS
+Documentation for 2022 Summer Internship with OSPO
+
+## Project Goal
+The summer project is a side project for [Sandiego](https://github.com/sandiego-rh/sandiego) and [Explorer](https://github.com/sandiego-rh/explorer). Project Sandiego is a front-end development that gleans information from open source ecosystem data sets that help drive community and business-oriented decision-making.
```
Comment (JamesKunstle) [inline models/density_metrics/README.md:5]: I would reword this project goal section.

Focus on being very direct with your organization and description. 

Drop the description of Sandiego and begin by describing the current gap (what this project isn't including yet). 

Then, describe what this project covers, e.g. :

"This project uses the Orbit Love model (ref here). We use relevant data about Github projects (PR, commit, user, and issue data) to create the following novel interpretations: (describe the following graphs that you put in your app). These visualizations might be used by a community architect or a community manager to more fully understand the contributor-ship trends of their projects.

Essentially, I think you ought to frame it as a more complete research project.

File: models/density_metrics/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+The goal for my summer project - density metrics, applied statistical methods to developing algorithms to quantify GitHub log data and user data on activity, community, and performance. 
+
+The metrics are based on activities in PR, issue, committer, etc for repositories within an organization. The goal of the project is to assist open-source community managers monitor and diagnosing community health and making actionable decisions.
+
+## Resources
+Useful metrics link:
+
+[CHAOSS](https://chaoss.community/)
+
+[ORBIT MODEL](https://orbitmodel.com/)
```
Comment (JamesKunstle) [inline models/density_metrics/README.md:16]: Since you rely on the work very heavily, I would copy and reference the creator's description of this model and include a thorough background in this README for one to understand what the model is trying to capture.

File: models/density_metrics/README.md (hunk 1 above)
Comment (yuunlee) [reply models/density_metrics/README.md:28]: I looked very closely on the site and unfortunately, I don't think the [Orbit Model](https://orbitmodel.com/activities) has described the reasoning behind their weight selection. Also, the site did mention that it is only a general recommendation. However, I do think weight optimization is something worth more effort, and perhaps some machine learning methods could be implemented in the future!

**Comment by cdolfi** [comment]
closing as other PR with changes was merged
